scripts/data_ingestion/ingest_espn_plays.py

```
@@ -0,0 +1,195 @@
+        if response.status_code == 200:
+            data = response.json()
+            checksum = compute_checksum(data)
+            
+            return {
+                "url": url,
+                "status": response.status_code,
+                "response_time_ms": response_time_ms,
+                "data": data,
+                "checksum": checksum,
+            }
+        else:
```

Ruff (0.15.10)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)